hi-ml-cpath/src/health_cpath/models/transforms.py

```diff
@@ -26,16 +26,33 @@ def load_pil_image(image_path: PathOrString) -> PIL.Image.Image:
     return image
 
 
-def load_image_as_tensor(image_path: PathOrString) -> torch.Tensor:
-    """Load an image as a tensor from the given path"""
+def load_image_as_tensor(image_path: PathOrString, scale_intensity: bool = True) -> torch.Tensor:
+    """Load an image as a tensor from the given path
+
+    :param image_path: path to the image
+    :param scale_intensity: if True, scale the intensity to [0, 1]. Otherwise, uses `to_tensor` from torchvision
+        which scales the image pixel intensities tp [0, 1] by default.
```

Comment:
Is this docstring the right way round? To me it looks like if scale_intensity is True, uses to_tensor, otherwise we return a transposed copy of the tensor.

Also there's nothing in the docs explaining why we transpose it/ permute the indices here?

Reply:
Good catch, my bad. I got confused in my own docs haha.

We need to permute because pil_image is in channel last format and we need channel first.
I will add a clarification comment